src/renderer/webtorrent.js

```diff
@@ -54,11 +54,16 @@ const VERSION_STR = VERSION.match(/([0-9]+)/g)
  */
 const VERSION_PREFIX = '-WD' + VERSION_STR + '-'
 
+/**
+ * Generate an ephemeral peer ID each time.
+ * TODO: once there are around 2^24 = ~8 million WebTorrent Desktops online at the same time,
+ *       ID collisions will start happening. Birthday paradox. Can we use more than six bytes?
```

Comment:
Btw, peer ID collisions aren't that bad... A malicious peer can already join a swarm, copy the peer id of an existing peer and start connecting to people. Worst thing that can happen is a mild DOS if peers refuse to connect to two peers with the same ID. But the attacker would actually have to be sending real torrent data to peers in order to remain connected and continue the DOS, so it's a pretty weak DOS, IMO.